public/index.html

```diff
@@ -36,14 +36,15 @@
     <script type="text/javascript" src="treeview.js"></script>
     <script type="text/javascript" src="js/lib/angular-animate.js"></script>
     <script type="text/javascript" src="js/toaster.js"></script>
-
+    <script type="text/javascript" src="js/ui-bootstrap-tpls-1.1.2.min.js"></script>
```

Comment:
I need to take out that line.  That was when I was monkeying around with the tabs using the ui-bootstrap.